### PR TITLE
Optimize some Chinese translations of the controld_control file

### PR DESCRIPTION
**Pull request by daaitudian**
Hi,
I have modified some Chinese translations to make them more fluent and understandable while retaining some proper nouns and symbols.

### CODE REVIEW

**Comment by kgaillot** [comment]
Hi, just to let you know, PR #3297 included some changes to option descriptions.

**Comment by daaitudian** [comment]
Thank you for your reminder. I will resubmit it after making modifications.

**Comment by kgaillot** [comment]
Hi @daaitudian , I will likely merge #3311 today which consolidates all the cluster options in a single table. Most of the descriptions remain unchanged. Once that's merged, you can redo these changes on the updated files. Hopefully that will be the last changes for the near future.

File: po/zh_CN.po [hunk 1]
```diff
@@ -184,30 +184,33 @@ msgid ""
...
 msgstr ""
-"如果设置为正值，则假定丢失的节点在这段时间内使用基于watchdog的SBD进行自我防"
-"护。这不需要明确配置fence资源，但可以配置一个fence_watchdog资源，以限制特定节"
-"点的使用。如果设置为0（默认值），集群将永远不会假定基于watchdog的自我防护。如"
-"果设置为负值，且如果`SBD_WATCHDOG_TIMEOUT`环境变量的本地值为正值，则集群将使"
-"用该值的两倍，否则将其视为0。警告：在使用基于watchdog的SBD的所有节点上，此超"
-"时必须大于`SBD_WATCGDOG_TIMEOUT`,如果本地值不是这样，或者SBD未运行，则"
-"Pacemaker将拒绝在任何节点上启动。如果设置为负值，则在使用SBD的所有节点上，"
-"`SBD_WATCHDOG_TIMEOUT`必须设置为相同的值，否则可能会发生数据损坏或丢失。"
+"这个值如果设置为正值, 丢失的节点将在设定的时间内将被假设使用基于 "
```
Comment (gao-yan) [inline po/zh_CN.po:187]: How about "正数" given that "负数" is used later?

"被认定" should be better than "被假设" here for "assumed".

File: po/zh_CN.po [hunk 2]
```diff
@@ -184,30 +184,33 @@ msgid ""
...
+"watchdog (watchdog-based)的SBD进行自我 fence (self-fencing)."
```
Comment (gao-yan) [inline po/zh_CN.po:188]: Similarly about "watchdog-based".

It's more precise with "完成了自我 fence" than "进行自我 fence".

File: po/zh_CN.po [hunk 3]
```diff
@@ -184,30 +184,33 @@ msgid ""
...
+"这不需要明确配置一个隔离(fence)资源, 但可以配置一个 fence_watchdog "
```
Comment (gao-yan) [inline po/zh_CN.po:189]: So far "fence" has remained literal without being translated everywhere. Probably we should keep it consistent here as well.

File: po/zh_CN.po [hunk 4]
```diff
@@ -184,30 +184,33 @@ msgid ""
...
+"资源来限制在特定节点上进行使用. 如果这个值设为0 (默认值), 集群将永远不会"
```
Comment (gao-yan) [inline po/zh_CN.po:190]: “来限制对特定节点使用”.

File: po/zh_CN.po [hunk 5]
```diff
@@ -184,30 +184,33 @@ msgid ""
...
+"假定节点使用基于 watchdog (watchdog-based)的自我 fence(self-fencing)"
```
Comment (gao-yan) [inline po/zh_CN.po:191]: To be consistent, "认定".

File: po/zh_CN.po [hunk 6]
```diff
@@ -184,30 +184,33 @@ msgid ""
...
+". 如果这个值设为负数, 集群将使用本地 `SBD_WATCHDOG_TIMEOUT` 环境变量"
+"的两倍值(如果该值为正), 否则会将该值视为0. 警告: 在所有使用基于 watchdog"
+"(watchdog-based)的SBD的节点上, 此超时值需大于`SBD_WATCHDOG_TIMEOUT`"
```
Comment (gao-yan) [inline po/zh_CN.po:194]: Similarly about "watchdog-based".

File: po/zh_CN.po [hunk 7]
```diff
@@ -184,30 +184,33 @@ msgid ""
...
+"的值, 否则Pacemaker将不会在任何不符合此条件的节点上启动. 当此值设为负数时"
```
Comment (gao-yan) [inline po/zh_CN.po:195]: We are missing the part: "也不会再任何未启用SBD的节点上运行"。

**Review by gao-yan — CHANGES_REQUESTED** [review]
Thanks. Some points in below.

**Comment by kgaillot** [comment]
@gao-yan , your review makes me realize the English wording is sometimes unclear on its own. If you want to point out where the English wording could be more specific and/or clear, I will come up with something that makes more sense, to make both English comprehension and translation easier

**Comment by daaitudian** [comment]
@kgaillot @gao-yan , Due to the current conflict, I have closed the current submission. I will submit a revised version based on your suggestions. Thanks for your help.

**Comment by gao-yan** [comment]
> @gao-yan , your review makes me realize the English wording is sometimes unclear on its own. If you want to point out where the English wording could be more specific and/or clear, I will come up with something that makes more sense, to make both English comprehension and translation easier

Makes perfect sense. Here we go: https://github.com/ClusterLabs/pacemaker/pull/3325

**Pull request closed without merging**